lmdeploy/model.py

```diff
@@ -1013,6 +1013,32 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
+@MODELS.register_module(name=['deepseek-chat'])
+class Deepseek(BaseModel):
+
+    def __init__(self, user='User:', assistant='Assistant:', **kwargs):
+        super().__init__(**kwargs)
+        self._user = user
+        self._assistant = assistant
+        self._bos = '<｜begin▁of▁sentence｜>'
```

Comment:
seems that the eos `<｜end▁of▁sentence｜>` is not used?

Reply:
No need